### DISCUSSION TITLE
Searched out Code does not have sort by

### DISCUSSION THREAD

**Discussion by a313132121** [community/community#60376] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

Searched out Code does not have sort by，Unable to find the latest

**Comment by look** [comment, marked as answer]
The ability to sort code by recency [was removed](https://github.blog/changelog/2023-03-10-changes-to-the-code-search-api/). There is another feedback discussion open about this: https://github.com/orgs/community/discussions/52932
